tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+skipsdist = True
+
+[testenv]
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+install_command = pip install {opts} {packages}
+
+[testenv:black]
+deps =
+  black >= 23.0, < 24.0
+
+commands =
+  black {toxinidir}/plugins {toxinidir}/tests
+
+[testenv:ansible-lint]
+deps =
+  ansible-lint==6.21.0
+changedir = {toxinidir}
+commands =
+  ansible-lint
+
+[testenv:linters]
+deps =
+  yamllint
```

Comment:
We can remove yamllint since we have ansible-lint.

Reply:
removed